src/shadowbox/server/manager_service.ts

```diff
@@ -69,9 +71,11 @@ export function bindService(
     apiServer: restify.Server, apiPrefix: string, service: ShadowsocksManagerService) {
   apiServer.put(`${apiPrefix}/name`, service.renameServer.bind(service));
   apiServer.get(`${apiPrefix}/server`, service.getServer.bind(service));
+  apiServer.put(`${apiPrefix}/default-access-key-port`, service.setDefaultPort.bind(service));
```

Comment:
Use `server/portForNewAccessKeys`, since the field is under server.

Reply:
I was thinking about this in the context of default quotas. `name` is a server property, yet its endpoint is '/name'. Do we want to be consistent with the default port/quota endpoints, or do we follow this style going forward?

Reply:
I didn't change this pending resolution of the conversation.  I don't care either way, this all seems like bikeshedding to me.

Reply:
Talked offline. Let's use `/server/port-for-new-access-keys`

Reply:
This still needs to change